docs/examples/plot_linear_gridsearch_tutorial.py

```diff
@@ -58,7 +58,7 @@
 ])
 
 ######################################################################
-# For the estimator ``MultiLabelEstimator``, arguments ``options`` is a LIBLINEAR option, and
+# For the estimator ``MultiLabelEstimator``, arguments ``options`` is a `LIBLINEAR option <https://github.com/cjlin1/liblinear/blob/70ecb8963b9390baf56892290b0d5a9e7e3d84ec/README#L99>`__, and
```

Comment:
why this weird URL address?  They need to read the whole section of train usage in README for understanding the meaning of options. Should have something like (see ?? in README at URL of liblinear github address)